pkgs/development/libraries/tevent/default.nix

```diff
@@ -20,7 +20,9 @@ stdenv.mkDerivation rec {
   '';
 
   configureFlags = [
-    "--bundled-libraries=NONE"
+    # "--bundled-libraries=NONE"
+    # ERROR: System library pytalloc-util of version 2.1.13 not found, and bundling disabled
+    "--bundled-libraries=pytalloc-util"
```

Comment:
Please switch tevent to Python 3 as described in https://github.com/NixOS/nixpkgs/issues/82489.

Reply:
Your PR is right and complete. I'll reduce this PR to contain only sssd's change.